src/record-error-archival/.webpack/webpack.common.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path")
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin")
+
+module.exports = {
+  entry: {
+    recordErrorArchival: "./src/index.ts"
+  },
+  resolve: {
+    modules: [path.resolve("./node_modules"), path.resolve("."), path.resolve("../shared/node_modules")],
+    extensions: [".js", ".json", ".ts"]
+  },
+  output: {
+    libraryTarget: "commonjs",
+    path: path.join(__dirname, "..", "build"),
+    filename: "[name].js"
+  },
+  externals: ['pino-pretty'],
+  target: "node",
+  module: {
+    rules: [
+      {
+        test: /\.(ts|js)?$/,
+        exclude: /node_modules/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: [
+              [
+                "@babel/preset-env",
+                {
+                  targets: {
+                    node: "14"
```

Comment:
It's worth targeting node 12, we've had issues transpiling to a version of 14 that conflicts with the lambda 14